lib/Marlin/Marlin/src/gcode/bedlevel/M420.cpp

```diff
@@ -87,12 +87,12 @@ void GcodeSuite::M420() {
     }
   #endif
 
+  const xyz_pos_t oldpos = current_position;
+
   // If disabling leveling do it right away
   // (Don't disable for just M420 or M420 V)
   if (seen_S && !to_enable) set_bed_leveling_enabled(false);
 
-  xyz_pos_t oldpos = current_position;
```

Comment:
Does have this change any effect?

Reply:
In al the unrelated changes I took the upstream resolution. In this case, oldpos was moved up and should have no effect. This was something I did as I manually introduced const to fix overload resolution, but I'm going to fix this by fixing types.h instead shortly.

Reply:
Removed this and another similar change in M420 which is no longer required.